x-pack/plugins/apm/public/components/shared/main_tabs.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiTabs } from '@elastic/eui';
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+// Since our `EuiTab` components have `APMLink`s inside of them and not just
+// `href`s, we need to override the color of the links inside or they will all
+// be the primary color.
+const StyledTabs = styled(EuiTabs)`
+  padding: ${({ theme }) =>
+    `${theme.eui.gutterTypes.gutterMedium} ${theme.eui.gutterTypes.gutterLarge}`};
```

Comment:
```suggestion
    `${theme.eui.gutterTypes.gutterMedium}`};
```

I reckon the padding shouldn't be larger on the sides to make it align down the page.

**Before**

<img width="592" alt="Screenshot 2020-11-10 at 09 23 51" src="https://user-images.githubusercontent.com/4104278/98648495-a813ad00-2336-11eb-8842-ba8e0b3abc03.png">

**After**

<img width="650" alt="Screenshot 2020-11-10 at 09 23 33" src="https://user-images.githubusercontent.com/4104278/98648513-acd86100-2336-11eb-87c0-872f659a600e.png">